homework/shelkovich_ilya/lookUpArrayService/src/main/java/com/tutrit/java/quickstart/service/LookUpArrayService.java

```diff
@@ -0,0 +1,72 @@
+package com.tutrit.java.quickstart.service;
+
+public class LookUpArrayService {
+
+
+    public int findMax(int[] array) {
+        if (array == null || array.length == 0) {
+            return 0;
```

Comment:
This is a verification logic. It has nothing to do with findMax logic, therefor could be encapsulated in separate method.

Reply:
created separate method to check is array empty